homeassistant/components/light/wink.py

```diff
@@ -12,10 +12,9 @@
 from homeassistant.components.wink import WinkToggleDevice
 from homeassistant.const import CONF_ACCESS_TOKEN
 
-REQUIREMENTS = ['https://github.com/balloob/python-wink/archive/'
-                '42fdcfa721b1bc583688e3592d8427f4c13ba6d9.zip'
-                '#python-wink==0.2']
-
+REQUIREMENTS = ['https://github.com/bradsk88/python-wink/archive/'
```

Comment:
Sorry about this, but with the recent changes Python Wink is kinda a moving target. This should be the last change:

As it is now published to PyPi you can replace it with:
```python
REQUIREMENTS = ['python-wink==0.3.1']
```